test/unit/org/apache/cassandra/io/sstable/SSTableReaderTest.java

```diff
@@ -247,6 +253,23 @@ public void testReadRateTracking()
 
         Util.getAll(Util.cmd(store, key).includeRow("0").build());
         assertEquals(2, sstable.getReadMeter().count());
+
+        // With persistence enabled, we should be able to retrieve the state of the meter.
+        sstable.maybePersistSSTableReadMeter();
+
+        UntypedResultSet meter = SystemKeyspace.readSSTableActivity(store.keyspace.getName(), store.name, sstable.descriptor.id);
+        // So this is the only issue atm; seems like it's empty
```

Comment:
nit: What does this comment mean? Looks like the assertion verifies that is't _not_ empty?

Reply:
Vestigial from interim debugging; removed.